examples/provider/config.yaml

```diff
@@ -1,7 +1,23 @@
+---
 apiVersion: cortex.crossplane.io/v1alpha1
 kind: ProviderConfig
 metadata:
-  name: provider-cortex
+  name: providerconfig-cortex
 spec:
-  tenantId: example-tenant
-  address: http://cortex.default.svc:9009
\ No newline at end of file
+  address: https://metricstore.abc.net
+  tenantId: tenant-example
+  credentials:
+    source: Secret
+    secretRef:
+      name: secret-name
+      namespace: crossplane-system
+      key: credentials
+---
+# example secret with credentials {"username": "your_username", "password": "your_password"} in base64 encode
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-name
+  namespace: crossplane-system
+data:
+  credentials: eyJ1c2VybmFtZSI6ICJ5b3VyX3VzZXJuYW1lIiwgInBhc3N3b3JkIjogInlvdXJfcGFzc3dvcmQifQ==
```

Comment:
What do you think about having two keys, `user` and `password`, instead of a json encoded one in `credentials`?

Reply:
this depends on how we will get the secret from AWS Secret Manager, so might be subject to change. As of now, the Service Catalog products create the secrets like that.

Reply:
I updated the PR with parameterization of secret keys that correspond to the cortex configurations for api user, key or auth token.